src/pages/Sketch/components/Canvas.js

```diff
@@ -100,29 +101,29 @@ function Canvas() {
         const col = hexToRGB(color);
         context.strokeStyle = `rgba(${col.red}, ${col.green}, ${col.blue}, 1)`;
 
-        if(stroke === 'small') {
-            context.setLineDash([5, parseInt(width)+3]);
-        } else if(stroke === 'big') {
-            context.setLineDash([5, parseInt(width)+10]);
+        if (stroke === 'small') {
+            context.setLineDash([5, parseInt(width) + 3]);
+        } else if (stroke === 'big') {
+            context.setLineDash([5, parseInt(width) + 10]);
         } else {
             context.setLineDash([]);
         }
-        
+
         context.lineJoin = edge;
         context.lineCap = 'round';
         context.lineWidth = width;
 
-        if(type === 'pen') {
+        if (type === 'pen') {
             logicDown(point);
-        } else if(type === 'line' || type === 'square' || type === 'circle' || type === 'triangle' || type === 'arrow' || type === 'diamond') {
+        } else if (type === 'line' || type === 'square' || type === 'circle' || type === 'triangle' || type === 'arrow' || type === 'diamond') {
```

Comment:
```suggestion
        } else if (['line', 'square', 'circle', 'triangle', 'arrow', 'diamond'].includes(type)) {
```
DO this instead.

Reply:
@smaranjitghose Don't you think this is the better way?

Reply:
Ok Sir